src/CoreWCF.Http/src/CoreWCF/Channels/TransportDefaults.cs

```diff
@@ -9,20 +9,21 @@ namespace CoreWCF.Channels
 {
     public static class TransportDefaults
     {
-        internal const long MaxReceivedMessageSize = 65536;
-        internal const int MaxBufferSize = (int)MaxReceivedMessageSize;
+        public const long MaxReceivedMessageSize = 65536;
+        public const int MaxBufferSize = (int)MaxReceivedMessageSize;
```

Comment:
This class shouldn't have been public, that was a mistake. There's no existing public members so nobody is consuming it. Leave the members as internal and copy the class with needed members into the new project. Same with HttpTransportDefaults.

Reply:
I've addressed this in 65884a453baec1859de29ee4c02d571181a5e78a. I did have to make TransportDefaults internal in Primitives as well.